README.md

```diff
@@ -1,5 +1,6 @@
 # OSPO Project Template
 
+![Release](https://img.shields.io/badge/OSS%20Template-0.3.1-brightgreen.svg)
```

Comment:
Should this link back to the original oss-template repo? main repo or the changelog, or diff between this version and the current?

Reply:
I like that idea! A link to the repo at a minimum, and a link to the diff if possible.